### ISSUE TITLE
How to debug config file

### ISSUE THREAD

**Issue by psamim** [sencer/i3-notifier#3]
Hi,

I tried playing with the config file and it seems sometimes there are errors and it crashes. But I could not find where to see the errors or the stack trace. I am debugging blindly.

Thanks for the good work

**Comment by sencer** [comment]
Hi,

Did you try starting `i3-notifier` with `--debug` flag? This will log stuff
to `/tmp/i3-notifier.log`, might be helpful.

**Comment by psamim** [comment]
@sencer Thanks for your reply. I used `--debug` and it does log everything to the log file, but it stops logging when there is an unexpected exception and the program crashes. The exceptions and stack traces are not logged.

**Comment by sencer** [comment]
You can add more debugging. Can you share the log file you get? I wonder what happens before the crash.

**Comment by psamim** [comment]
@sencer I found the bug (I had not imported a module), but I was wondering if I can run it _not_ as a daemon to see the stack traces when it crashes.

**Comment by sencer** [comment]
I don't think you can as is. Should be easy to modify `bin/i3-notifier` though.
